cmd/catalyst-node/handlers/access-control/access-control.go

```diff
@@ -102,24 +105,19 @@ func (ac *PlaybackAccessControl) handleUserNew(payload []byte) []byte {
 
 	var pubKey string
 	if jwtToken != "" {
-		claims, err := decodeJwt(jwtToken)
-		if err != nil {
-			glog.Errorf("Unable to decode on incoming playbackId=%v jwt=%v", playbackID, jwtToken)
-			return []byte("false")
-		}
-
-		if playbackID != claims.Subject {
-
-			glog.Errorf("PlaybackId mismatch playbackId=%v != claimed=%v", playbackID, claims.Subject)
+		pubKey = extractKeyFromJwt(jwtToken, playbackID)
```

Comment:
WDYT of just using `IsAuthorized` helper with the `requestURL` var as an arg? Then we get `accessKey` auth for livestreams as well! (it seems like we don't yet, right?)

Reply:
Ah sorry ha, I looked at it initially and just thought this was totally separate as it's dealing with some POSTed payload, but we get a url out of that payload so it's fine :) will fix that!

Reply:
@victorges what do you think about making playbackID a param for the `IsAuthorized` func? I'm just a bit concerned about assuming the url structure and extracting it from the path, seems a bit error prone and any time there's a new url format we'd have to adjust the function.

Reply:
Sure I think that makes sense to me as well!